python/ray/actor.py

```diff
@@ -612,22 +681,57 @@ def _actor_method_call(self,
 
         args = args or []
         kwargs = kwargs or {}
-        function_signature = self._ray_method_signatures[method_name]
-
-        if not args and not kwargs and not function_signature:
-            list_args = []
+        if self._ray_is_cross_language:
+            if not worker.load_code_from_local:
+                raise Exception("Cross language feature needs "
+                                "--load-code-from-local to be set.")
+            if self._ray_actor_language == Language.PYTHON:
+                # To compatible with ray.signature.recover_args
+                function_signature = signature.ANY_FUNCTION_SIGNATURE
+                list_args = signature.flatten_args(function_signature, args,
+                                                   kwargs)
+            else:
+                if kwargs:
+                    raise Exception("Cross language remote actor method "
+                                    "not support kwargs.")
```

Comment:
there seems to be duplicates above, could these be moved into a separate function?